Tools/CISupport/ews-build/config.json

```diff
@@ -350,6 +350,7 @@
       "type": "AnyBranchScheduler", "name": "pullrequest",
       "builderNames": [
             "Bindings-Tests-EWS", "iOS-15-Build-EWS",
+            "macOS-AppleSilicon-Big-Sur-Debug-Build-EWS", "macOS-BigSur-Debug-Build-EWS", "macOS-BigSur-Release-Build-EWS",
```

Comment:
For now, let's enable either "macOS-AppleSilicon-Big-Sur-Debug-Build-EWS" or "macOS-BigSur-Debug-Build-EWS" and see how it goes.

Reply:
To follow-up on this discussion, I enabled `macOS-AppleSilicon-Big-Sur-Debug-Build-EWS` this morning without committing the change. Everything has gone well, I think we should enable all Mac queues tomorrow morning.